cognite/extractorutils/configtools.py

```diff
@@ -191,6 +192,7 @@ class LoggingConfig:
 
     console: Optional[_ConsoleLoggingConfig]
     file: Optional[_FileLoggingConfig]
+    log_metrics: Optional[bool] = False  # enables metrics on the number of log messages recorded (per logger and level)
```

Comment:
I'm not sure it's the best name for the config parameter if you have better ideas, shoot!

Reply:
It makes sense, after reading the comment